### DISCUSSION TITLE
Is there an open-source design kit for Auro Design System?

### DISCUSSION THREAD

**Discussion by ParishrutB** [AlaskaAirlines/WC-Generator#569] [General Auro discussions]
Hi everyone, 

I have been searching for a design kit for Auro. I saw that some components like Button had a design kit in Figma, while others needed some permissions. 

Is there an open-source version of the Auro Design System, in Figma, which contains all components?

**Comment by ryanscottalaska** [comment]
Hi 'ParishrutB' - Lindsey O'Sullivan brought this question to my attention. I'm not on GitHub often. If you could send me a Teams Message that would be great. I'm Ryan Scott. I can show you were to access these items in Figma. In general the built-in Library has all the elements you need. There is no one-page 'kit' anymore. But I can show you how to access everything you need. - Ryan
